ckanext/xloader/templates/package/snippets/resources.html

```diff
@@ -0,0 +1,8 @@
+{% ckan_extends %}
+
+{% block resources_list_edit_dropdown_inner %}
```

Comment:
This block exists only in CKAN core master, not in any released version, and so it will have no effect.

Reply:
yeah there is no block in less than 2.10 that does anything in this template

Reply:
Ah, my bad, I missed the fact that this block appears in the Datastore templates rather than core ones. So it's 2.10 only but otherwise okay.